src/appengine/libs/issue_management/jira/__init__.py

```diff
@@ -0,0 +1,196 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Jira issue tracker."""
+
+from libs.issue_management import issue_tracker
+from libs.issue_management.jira.issue_tracker_manager import (
+    IssueTrackerManager)
+
+
+class Issue(issue_tracker.Issue):
```

Comment:
Nit: we use two spaces for indentation, and it looks like our lint checks found a few other issues. Could you look through the travis logs and fix the issues it reported? You can also run our lint (and a few other) checks locally with "python butler.py lint".

Reply:
Will do